<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>901</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Real Input</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>290</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Real Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_real">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>50</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_real">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>320</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calcButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>710</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_imag">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>50</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_imag">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>320</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="randomButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>850</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputLabel_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Imaginary Input</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Imaginary Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sizeInput">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>780</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="stepsInput">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>780</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="graphTypeComboBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>560</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
